rest-api/dao/participant_summary_dao.py

```diff
@@ -233,7 +233,8 @@ def update_from_biobank_stored_samples(self, participant_id=None):
                                 'received': int(SampleStatus.RECEIVED),
                                 'full_participant': int(EnrollmentStatus.FULL_PARTICIPANT),
                                 'member': int(EnrollmentStatus.MEMBER),
-                                'interested': int(EnrollmentStatus.INTERESTED)}
+                                'interested': int(EnrollmentStatus.INTERESTED),
+                                'lastModified': clock.CLOCK.now()}
```

Comment:
This isn't referenced in the SQL?

This one is going to be tricky... there is no WHERE clause on this update and so if we didn't do anything you'd update lastModified on all rows every time this runs.

Instead of using clock.now, we probably(?) need to add some SQL that will set lastModified to the MAX of (lastModified, biobank stored sample timestamps), such that nothing changes for participants where the other columns didn't change.

Reply:
done